src/blockchain_db/blockchain_db.h

```diff
@@ -160,15 +160,14 @@ struct txpool_tx_meta_t
   uint64_t max_used_block_height;
   uint64_t last_failed_height;
   uint64_t receive_time;
-  uint64_t last_relayed_time;
+  uint64_t last_relayed_time; //!< If received over i2p/tor, randomized forward time. Otherwise, last relayed timestamp
   // 112 bytes
-  uint8_t kept_by_block;
+  uint8_t state;
   uint8_t relayed;
   uint8_t do_not_relay;
   uint8_t double_spend_seen: 1;
   uint8_t pruned: 1;
```

Comment:
This drops a mempool meta bit flag that has never been in a release, but has been in `master`. I realized it was more efficient to use `kept_by_block` instead. Anyone running from `master` that has a `is_local` flag in their pool will see incorrect behavior when running this change. I think its better to clean this up before a release, but ... ?

Reply:
Are you saying kept_by_block is new ? It's not. Or are you talking about something else here ?